gravitee-am-identityprovider/gravitee-am-identityprovider-api/src/main/java/io/gravitee/am/identityprovider/api/oidc/OpenIDConnectConfigurationUtils.java

```diff
@@ -40,4 +41,21 @@ public static Optional<String> extractCertificateId(String configuration) {
             return Optional.empty();
         }
     }
+
+    public static String validateClientAuthMethod(String configuration) {
```

Comment:
**suggestion**: Looking at the method behaviour it more a _sanitizeClientAuthMethod_ than _validateClientAuthMethod_

```suggestion
    public static String sanitizeClientAuthMethod(String configuration) {
```

Reply:
I changed it.